src/components/Slider/__test__/Slider.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+import Slider from "../Slider";
+import { SLIDER_IMAGES } from "../../../common/constants";
+
+const renderComponent = async () => {
+  let result;
+
+  await waitFor(async () => {
+    result = render(<Slider />);
+  });
+
+  return result;
+};
+
+describe("Slider component", () => {
```

Comment:
It would be nice to add a unit test that the slider component is on the page. For example, you can find a slider wrapper by test id